IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -102,6 +105,37 @@ - (id)initWithStartURL:(NSURL *)startURL
     return self;
 }
 
+#if MSAL_JS_AUTOMATION
+- (id)initWithStartURL:(NSURL *)startURL
+                endURL:(NSURL *)endURL
+               webview:(WKWebView *)webview
+         customHeaders:(NSDictionary<NSString *, NSString *> *)customHeaders
+        platfromParams:(MSIDWebViewPlatformParams *)platformParams
+            javascript:(NSString *)javascript
+               context:(id<MSIDRequestContext>)context
+{
+    
+    self = [super initWithContext:context
+                   platformParams:platformParams];
+
+    if (self)
+    {
+        self.webView = webview;
```

Comment:
we have similar logic in another initializer. Should we extract common logic into a separate method?

Reply:
@antrix1989 I'm not sure about this. In my opinion, I feel it's weird to extract such initialization code into a separate method. And in most of the cases, this will be ignored (only useful while building the broker test app for MSALCPP). Therefore, I think it's better to keep this.

Reply:
This is a good practice to have one "designated initializer" and other "convenient" initializers. If we need to add new property in this class in future, most probably we will forget to update your test initializer. Your tests will start failing or even worse -- tests won't catch the potential bug in the code. That is why I still suggest to extract shared logic.

Reply:
That's a good point. I'll make the change.

Reply:
I think I can just reused the base initializer code.